From 40dd4a1e7d788f85be1ad07da4050c94a7f786ec Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Thu, 17 Oct 2019 10:05:50 +0000
Subject: [PATCH] MANTA-3101 mako nginx logs can be uploaded under wrong path
 Reviewed by: Robert Bogart <Robert.bogart@joyent.com> Reviewed by: Trent Mick
 <trentm@gmail.com> Approved by: Trent Mick <trentm@gmail.com>

---
 boot/setup.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/boot/setup.sh b/boot/setup.sh
index 0355f3d..61bec8c 100644
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -7,7 +7,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 set -o xtrace
@@ -129,7 +129,7 @@ function manta_setup_nginx {
 
     logadm -w mako_logs -C 48 -p 1h \
         -t '/var/log/manta/upload/$basename_$nodename_%FT%H:00:00.log' \
-        -a 'pkill -USR1 -ox nginx; cd /var/log/manta/upload; mv mako-error* $(ls mako-error* | sed "s/\.log_/_/"); mv mako-access* $(ls mako-access* | sed "s/\.log_/_/")' \
+        -a 'pkill -USR1 -ox nginx; cd /var/log/manta/upload; for file in mako-error.log_* mako-access.log_*; do mv "$file" "${file/.log_/_}"; done' \
         '/var/log/mako-{access,error}.log'
 }
 
-- 
2.17.2 (Apple Git-113)

